src/frontend/screens/DeviceNaming/GetStarted.tsx

```diff
@@ -0,0 +1,70 @@
+import * as React from 'react';
+import {StyleSheet, View} from 'react-native';
+import {Text} from '../../sharedComponents/Text';
+import TopoLinesBackground from '../../images/TopoLinesTransparent.svg';
+import CoMapeoLogo from '../../images/CoMapeoLogo.svg';
+import {defineMessages, useIntl} from 'react-intl';
+import {Button} from '../../sharedComponents/Button';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {DeviceNamingList} from '../../Navigation/ScreenGroups/DeviceNamingScreens';
+
+const m = defineMessages({
+  startMessage: {
+    id: 'screens.DeviceNaming.GetStarted.startMessage',
+    defaultMessage: 'Collaborate on mapping and monitoring projects',
+  },
+  getStarted: {
+    id: 'screens.DeviceNaming.GetStarted.getStarted',
+    defaultMessage: 'Get Started',
+  },
+});
+
+const TopoLinesWidth = 360;
+const TopoLineHeight = 640;
```

Comment:
Mostly interested in adding `original` somewhere, to make it clearer about they're representing. the screaming snakecase is just personal preference

```suggestion
const TOPO_LINES_SVG_ORIGINAL_WIDTH = 360;
const TOPO_LINES_SVG_ORIGINAL_HEIGHT = 640;
```